### ISSUE TITLE
It seems supervisor cann't delay a process to get started

### ISSUE THREAD

**Issue by zzygit** [Supervisor/supervisor#954]
I have 2 programs. The first one name P1 and the second one is P2. P2 is depend on P1, I want P1 be started completely first, and then start P2. I can not find a way to config supervisor working like this.

**Comment by julien6387** [comment]
Hi zzy,

I have implemented this kind of dependency in a Supervisor plugin:
https://github.com/julien6387/supvisors

More particularly, you could have a look at:
http://supvisors.readthedocs.io/en/0.1b5/configuration.html#program-rules
http://supvisors.readthedocs.io/en/0.1b5/special.html#starting-strategy
http://supvisors.readthedocs.io/en/0.1b5/special.html#stopping-strategy

If you have any questions, feel free to contact me on my github profile.


2017-06-22 9:41 GMT+02:00 zzy <notifications@github.com>:

**Comment by mnaberez** [comment]
Duplicate of #122.
